ext/js/background/backend.js

```diff
@@ -2472,6 +2472,7 @@ export class Backend {
         if (mode === 'merge' && !enabledDictionaryMap.has(mainDictionary)) {
             enabledDictionaryMap.set(mainDictionary, {
                 index: enabledDictionaryMap.size,
+                alias: mainDictionary,
```

Comment:
Should this maybe be the mainDictionary's alias instead of it's name?

Reply:
https://github.com/FooSoft/yomichan/pull/1729

Looks like this is used for grouping using main dictionary, but excluded all the results from the main dictionary. Basically, the main dictionary here will never be shown on popup so the alias does not matter here.